api/src/assets/validator-handle.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
It would be better if this data were dynamic, loaded into the database or similar, and not in the git repo.

Reply:
done. thanks

Reply:
please add the env var:
```
VALIDATOR_HANDLES_URL="https://raw.githubusercontent.com/0LNetworkCommunity/v7-addresses/refs/heads/main/validator-handle.json"```